views/components/periodSelect.njk

```diff
@@ -0,0 +1,22 @@
+<div class="pure-u-md-1-6">
```

Comment:
should be a https://design-system.service.gov.uk/components/select/

Reply:
It's not required anyway.  I updated it to use the grid layout.

Reply:
But yes.  I need to get my head around the GDS stuff.

Reply:
Actually, i'll update with the appropriate classes.